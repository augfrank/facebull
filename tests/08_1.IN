M0   C0   C4  2270
M1   C1   C0  2749
M2   C2   C6  2861
M3   C2   C7  2666
M4   C3   C1  2259
M5   C4   C0  2817
M6   C4   C2  2826
M7   C4   C5  2739
M8   C5   C1  2181
M9   C5   C3  2562
M10   C6   C5  2300
M11   C7   C5  2332
M12   C7   C6  2110
