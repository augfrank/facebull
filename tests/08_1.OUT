19742
0 1 3 4 6 9 10 12
